doc/development.rst

```diff
@@ -37,6 +37,212 @@ The source repository (and current branch) of each of these packages
 in use is determined by the file
 `requirements.txt in mirgecom <https://github.com/illinois-ceesd/mirgecom/blob/master/requirements.txt>`__.
 
+
+Working with Pull Requests
+--------------------------
+
+We are using GitHub's pull requests (PRs) feature to integrate changes into
+mirgecom and its supporting packages. Pull requests are based on git branches
+that are merged into another branch (usually the master branch). Note that
+pull requests are a GitHub feature and live outside the main git
+functionality; the ``git`` program itself has no knowledge of them.
+
+.. note::
+
+   A good overview of basic git functionality and the ``git`` program can be found
+   on this `GitHub page <https://try.github.io/>`__.
+
+.. note::
+
+   On most systems, the ``git`` program will already be installed. You can
+   install it yourself by running ``conda install git``.
+
+Forking the repository
+^^^^^^^^^^^^^^^^^^^^^^
+
+In most cases, it will be necessary to fork the repository (that is, create a
+copy of the repository in your own GitHub account), since creating a pull
+request requires creating a branch in the repo, which requires write access.
+You can fork a repository directly on the GitHub website.
```

Comment:
This should cover adding a remote for the forked repo. (Used under the name `my_fork` below.)

Reply:
Good idea, done.